frontend/src/pages/InsideStudyMode.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import Header from "../main-components/Header";
+import Sidebar from "../main-components/Sidebar";
+import StudyModeContent from "../main-components/StudyModeContent";
+import css from "./template.css";
+
+function InsideStudyMode(props) {
+  // Handles mobile menu button and updates sidebar view
+  const [sideSetting, setSideSetting] = useState("f");
+  const handleClick = (e) => {
+    if (sideSetting === "f") {
+      setSideSetting("t");
+    } else {
+      setSideSetting("f");
+    }
```

Comment:
When depending on the previous value of state, it's best to use the functional setState:

```js
setSideSetting(sideSetting => !sideSetting);
```

Reply:
done.